src/core/api_client/tests/factories/mock.factory.ts

```
@@ -42,10 +45,12 @@ export class MockFactory {
   }
 
   static setupLogoutEndpoint(config: AuthConfig = {}): void {
-    const { baseUrl, realm } = AuthFactory.createConfig(config);
-    const logoutEndpoint = `${baseUrl}/realms/${realm}/protocol/openid-connect/logout`;
-    fetchMock.post(logoutEndpoint, {
-      status: 204,
+    const logoutEndpoint = AuthFactory.getLogoutEndpoint(config);
+    fetchMock.post(logoutEndpoint, (url: string, opts: any) => {
+      this.callCount++;
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)